src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/matmul.cpp

```diff
@@ -231,8 +57,11 @@ const std::vector<ElementType> netPRCs {
 
 std::vector<CPUSpecificParams> filterSpecificParams() {
     std::vector<CPUSpecificParams> specificParams;
+#if defined(OPENVINO_ARCH_ARM64) || defined(OPENVINO_ARCH_ARM)
```

Comment:
Can we create a filter function which will filter out extra implementations for common scope.
So we insert both "acl" and "jit_gemm" info the params list, but the filter function will remove extra params depending on the architecture.

Reply:
filterCpuInfo() {
    filterCpuInfoForArch()
    filterCPUInfoForDevice()
}

Reply:
implemented